### ISSUE TITLE
論文マラソン３週目

### ISSUE THREAD

**Issue by rild** [rild/TIL#6]

**Comment by rild** [comment]
- 3-1 
WaveNet における音声波形量子化法の評価 ∗ ◎橘健太郎 (NICT), 戸田智基 (名大/NICT), 志賀芳則, 河井恒 (NICT)

日本音響学会春季大会

**Comment by rild** [comment]
条件付き確率

**Comment by rild** [comment]
- 3-2 https://arxiv.org/abs/1601.06759
Pixcel Recurrent Neural Networks 

WaveNet の元の手法らしい

> 誤差関数が異なる.
二乗誤差ではなく, softmax に基づく, cross entropy を用いている.
- ガウス分布を仮定した平均ベクトルの誤差最小問題
が,
- 音声信号の振幅値の多クラス分類問題
に置き換えることができるようになる.

**Comment by rild** [comment]
- 3-3 Deepvoice
https://arxiv.org/pdf/1702.07825.pdf

liked from 
http://sergeiturukin.com/2017/03/22/training-experiments.html

**Comment by rild** [comment]
- 3-4 DNNを用いた聴覚障害者の音声合成の検討* ☆北村毅(神戸大)，滝口哲也(神戸大/JST さきがけ)，有木康雄(神戸大)

ASJSpring2017

**Comment by rild** [comment]
- 3-5 ResNet 
[Cornell Uni Lib](https://arxiv.org/abs/1512.03385)

**Comment by rild** [comment]
- 3-6 The first type is the Row LSTM layer

where the convolution is applied along each row

[Cornel Uni Lib](https://arxiv.org/abs/1506.07452)

**Comment by rild** [comment]
- 3-7 JP-EN ss VOCALOID – Commercial singing synthesizer based on sample concatenation 
[pdf link](https://ai2-s2-pdfs.s3.amazonaws.com/4606/eeaa5945a262639879cc8bd00b1cadd39a86.pdf)

**Comment by rild** [comment]
- 3-8 STATISTICAL PARAMETRIC SPEECH SYNTHESIS USING DEEP NEURAL NETWORKS
[pdf link](https://research.google.com/pubs/pub40837.html)

deepvoice reference

**Comment by rild** [comment]
- XX 
海藻ｲｹﾙﾜｧ
http://www.nature.com/nature/journal/v464/n7290/abs/nature08937.html

**Comment by rild** [comment]
- 3-9 Architectural Complexity Measures of Recurrent Neural Networks

Twitter で[回ってきた](https://twitter.com/ballforest/status/858880173688487936)
[NIPS2016 / Architectural Complexity Measures of Recurrent Neural Networks](https://papers.nips.cc/paper/6303-architectural-complexity-measures-of-recurrent-neural-networks)

**Comment by rild** [comment]
くっ
